tests/unit/models/TerraformData.spec.js

```diff
@@ -0,0 +1,194 @@
+import TerraformComponentAttribute from 'src/models/TerraformComponentAttribute';
+import TerraformComponentAttributeDefinition
+  from 'src/models/TerraformComponentAttributeDefinition';
+import TerraformData from 'src/models/TerraformData';
+import TerraformVariable from 'src/models/TerraformVariable';
+import { Component } from 'leto-modelizer-plugin-core';
+
+describe('Test class: TerraformData', () => {
+  const terraformData = new TerraformData();
+
+  describe('Test method: getVariableValue', () => {
+    it('Should return the value of a variable', () => {
+      terraformData.variables.push(new TerraformVariable({
+        name: 'test_variable1',
+        value: 'test_value',
+        category: 'variable',
+      }));
+      terraformData.variables.push(new TerraformVariable({
+        name: 'test_variable2',
+        value: null,
+        category: 'variable',
+      }));
+
+      expect(terraformData.getVariableValue('var.test_variable1'))
+        .toEqual('test_value');
+      expect(terraformData.getVariableValue('var.test_variable2'))
+        .toBeNull();
+    });
+
+    it('Should return the value of a local variable', () => {
+      terraformData.variables.push(new TerraformVariable({
+        name: 'test_variable1',
+        value: 'test_value',
+        category: 'local',
+      }));
+      terraformData.variables.push(new TerraformVariable({
+        name: 'test_variable2',
+        value: null,
+        category: 'local',
+      }));
+
+      expect(terraformData.getVariableValue('local.test_variable1'))
+        .toEqual('test_value');
+      expect(terraformData.getVariableValue('local.test_variable2'))
+        .toBeNull();
+    });
+
+    it('Should return null on bad variable reference', () => {
+      const variable = new TerraformVariable({
+        name: 'test_variable',
+        value: 'test_value',
+        category: 'variable',
+      });
+
+      terraformData.variables.push(variable);
+      expect(terraformData.getVariableValue('var.test_variable.error'))
+        .toBeNull();
+    });
+  });
+
+  describe('Test method: getAttributeValue', () => {
+    it('Should return the value of a variable', () => {
+      const variable = new TerraformVariable({
+        name: 'test_variable',
+        value: 'test_value',
+        category: 'variable',
+      });
+      const attribute = new TerraformComponentAttribute({
+        value: 'var.test_variable',
+      });
+
+      terraformData.variables.push(variable);
+      expect(terraformData.getAttributeValue(attribute))
+        .toEqual('test_value');
+    });
+
+    it('Should return the ID of components links', () => {
```

Comment:
`it('Should return an array containing the ID of components links', () => {`

Reply:
done